config/initializers/settings_validator.rb

```diff
@@ -58,7 +58,11 @@ def print_errors
   # :nocov:
 
   # Checker methods
+  # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/MethodLength, Metrics/PerceivedComplexity
   def check_server_url
```

Comment:
This method might be worth to refactor (given how many rules it violates), however I didn't have a good alternative, so I just disabled the failing checks for now.